recipes-nodes/haproxy/haproxy.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "HAProxy"
+LICENSE = "HAPROXY's license - 2006/06/15"
```

Comment:
@MoeMahhouk it is literally the [name](https://github.com/haproxy/haproxy/blob/master/LICENSE) of the license. But since don't build HAProxy from source we don't embed the license file and can't use [LIC_FILES_CHKSUM](https://docs.yoctoproject.org/ref-manual/variables.html#term-LIC_FILES_CHKSUM). Wdyt?

Reply:
hmmm, I see you are not making a recipe to build it but rather to execute it directly in a containerized way. For that I have two suggestions here:
1- either make the LICENSE="closed" since the recipe is all made by you and it is executing HaProxy through a podman container image instead. 
2- Copy the license file here too and add it like this
```
LICENSE = "HAPROXY"
LIC_FILES_CHKSUM = "file://path/to/license/file;md5=<checksum>"
```
you can measure the sum of the file using md5sum <HaProxy-License>

Reply:
Added a LICENSE file + checksum